vnext/Microsoft.ReactNative/Fabric/Composition/CompositionRootView.cpp

```diff
@@ -288,8 +288,12 @@ void CompositionRootView::ShowInstanceLoaded() noexcept {
 
     auto rootTag = ::Microsoft::ReactNative::getNextRootViewTag();
     SetTag(rootTag);
-    uiManager->startSurface(
-        *this, rootTag, JSComponentName(), DynamicWriter::ToDynamic(Mso::Copy(m_reactViewOptions.InitialProps())));
+    auto initProps = DynamicWriter::ToDynamic(Mso::Copy(m_reactViewOptions.InitialProps()));
+    if (initProps.isNull()) {
+      initProps = folly::dynamic::object();
+    }
+    initProps["concurrentRoot"] = true;
```

Comment:
Totally up to you, but if you want to be consistent with mobile, you may want to put this in a quirk setting to disable concurrent root. I think it's on by default in mobile, but it can be explicitly disabled.

Reply:
Is it something that is settable per rootview on mobile?  Should I make it a prop of the rootview?

Reply:
On iOS, it's per AppDelegate (but I don't know if AppDelegates are per root):
https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm#L62

On Android, it appears to be per app, but inseparable from Fabric in general (perhaps):
https://www.internalfb.com/code/fbsource/[c1f78b3779b16717460c303260b0a975133a4321]/xplat/js/react-native-github/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java?lines=345